tests/notebooks/test_run_notebooks.py

```diff
@@ -1,54 +1,295 @@
 import os
 
+import nbformat
 import pytest
 from pyomo.common.fileutils import this_file_dir
 from testbook import testbook
 
 from omlt.dependencies import keras_available, onnx_available
 
+# TODO: We need to try and write these tests to rely on internal consistencies and less on absolute numbers and tolerances
 
-# TODO: These will be replaced with stronger tests using testbook soon
-def _test_run_notebook(folder, notebook_fname, n_cells):
-    # change to notebook directory to allow testing
-    cwd = os.getcwd()
+
+# return testbook for given notebook
+def open_book(folder, notebook_fname, **kwargs):
+    execute = kwargs.get("execute", True)

```

Comment:
Can you use named arguments instead of **kwargs in these methods? Actual arguments provide better documentation for someone using the functions.